src/scenes/home/header/topNav/topNav.css

```diff
@@ -1,8 +1,10 @@
 .topNav {
   display: flex;
+  margin-left: 400px;
```

Comment:
There should be a better way to center this text, other than using hardcoded px numbers and margin-left.  Maybe look at using flexbox properties and adjusting the heirarchy of layout elements?